R/utils.R

```diff
@@ -1,4 +1,8 @@
 as_rounded_int <- function(x) {
+  x_name <- deparse(substitute(x))
+  if (!is.numeric(x)) {
+    stop("`", x_name, "` must be an integer number of pixels, not '", x, "'.")
+  }
```

Comment:
```suggestion
  if (!is.numeric(x)) {
    x_name <- deparse(substitute(x))
    stop("`", x_name, "` must be a number representing the number of pixels, not '", x, "'.")
  }
```

Better to do the `substitute` stuff only when needed. Also, the input doesn't have to be an integer (the purpose of this function is to turn numbers into integers).

Reply:
I could move the check into `webshot2` but the point is that users should be providing integers for `vwidth` and `vheight`.

Reply:
```suggestion
  if (!is.numeric(x)) {
    x_name <- deparse(substitute(x))
    stop("`", x_name, "` must be an integer number of pixels, not '", x, "'.")
  }
```

Reply:
I could also rename this function now that it's more specific than just rounding integers